cmake/globalconfig.cmake

```diff
@@ -230,6 +230,28 @@ endif()
 # Mac specific
 if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     add_definitions(-DDARWIN)
+
+    # Set RPATH (relative path) differently depending on build-time and install/package.
+    # At build time we use full paths to the generatated libraries.
+    # When packaging, however, we must use relative paths so that the app 
+    # can be used on other computers.
+    # See:
+    # https://gitlab.kitware.com/cmake/community/-/wikis/doc/cmake/RPATH-handling#always-full-rpath
+    set(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
```

Comment:
does this replace the work that fixup_bundle did before? That did the rpath rewriting before..

Reply:
I think that fixup_bundle was only done on qt plugins before. But yes, it ensures that the RPATH for installed binaries are adjusted.